### DISCUSSION TITLE
Old password isn't valid - How to change

### DISCUSSION THREAD

**Discussion by chrisfinazzo** [community/community#23506] [New to GitHub]
<p>I want to delete my fork of a project as it is now inactive. However, I have enabled 2FA and use a Personal Access Token to push commits from the command-line.</p>
<p>When I attempt to change the password while logged in on the web, I get a response back that the old password stored in the macOS Keychain is incorrect.</p>
<p>What is the procedure to disable 2FA and reset the CLI token so that operations on repos and login work correctly again? Can I delete the Keychain item and then create a new one when the password is changed?</p>
<p>(I am hesitant to log out at this point as this may lock me out of my account.)</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="chrisfinazzo" data-post="1" data-topic="251261">
<div class="title">
<div class="quote-controls"></div>
 chrisfinazzo:</div>
<blockquote>
<p>What is the procedure to disable 2FA and reset the CLI token so that operations on repos and login work correctly again?</p>
</blockquote>
</aside>
<p>I’m a bit confused what you’re trying to do, because these are separate things. You can’t log in with your PAT, you need your account password and the configured second factor (TOTP, hardware token, or SMS). Does the account password not work like you think it should, or the token?</p>
<p>The most important thing to make sure you don’t get locked out: Do you have the recovery codes for your account? You can use those instead of the second factor in case you lose it.</p>

**Comment by chrisfinazzo** [comment]
<p>At this point, I honestly can’t remember what repo and activity caused the PAT to come into existence and it has been some time since I’ve had the need to delete a repo from the web side.</p>
<p>The PR associated with the fork was closed over the past week and I did not see the notification until this morning when I was fetching the upstream changes from all of the forked repos I track.</p>
<p>I do have recovery codes available - will try this to see if I can reset things and report back.</p>
<p><strong>Update</strong>: Resolved, thanks.</p>
<p><strong>Update <span class="hashtag">#2</span></strong>: Sigh, now I remember. I’m fairly certain the PAT is related to a <a href="https://github.com/getindiekit/indiekit" rel="noopener nofollow ugc">Indiekit</a> server and associated Twitter application which will syndicate items from one of my repo’s RSS feeds. In this instance, GitHub is a “content store” for the site’s data and deploys to my host (Netlify) are done when I push changes.</p>
<p>This is a WIP, so it’s kind of broken at the moment.</p>
